map/station.php

```diff
@@ -43,89 +43,45 @@
-		<link rel="icon" type="image/png" sizes="16x16" href="./images/favicon-16x16.png">
-
-		<link href="css/all.min.css" rel="stylesheet">
-		<link href="css/style.css" rel="stylesheet">
-
+<head>
+	<meta property="og:image" content="<?php echo htmlspecialchars($stationPhoto); ?>"/>
+	<title><?php echo htmlspecialchars($stationName); ?> - RailwayStations</title>
+	<?php require_once("./header.php") ?>
 </head>
 <body>
 
-	<nav class="navbar navbar-expand-md navbar-dark fixed-top" style="background-color: #9F0C35;">
-		<a class="navbar-brand" href="index.html" active>
-	    <img src="images/logo_white.svg" width="30" height="30" class="d-inline-block align-top" alt="">
-	    Railway<strong>Stations</strong>
-	  </a>
-		<button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
-	    <span class="navbar-toggler-icon"></span>
-	  </button>
-
-	  <div class="collapse navbar-collapse" id="navbar">
-	    <ul class="navbar-nav mr-auto">
-			<li class="nav-item">
-		      <a class="nav-link p-2" href="settings.html" rel="noopener" aria-label="Einstellungen" title="Einstellungen"><i class="fas fa-sliders-h"></i></a>
-		    </li>
-			<li class="nav-item">
-		      <a class="nav-link p-2" href="faq.html" rel="noopener" aria-label="FAQ" title="FAQ"><i class="fas fa-question"></i></a>
-		    </li>
-			<li class="nav-item">
-		      <a class="nav-link p-2" href="https://github.com/RailwayStations" rel="noopener" aria-label="Entwicklung" title="Entwicklung"><i class="fab fa-github"></i></a>
-		    </li>
-			<li class="nav-item">
-		      <a class="nav-link p-2" href="impressum.html" rel="noopener" aria-label="Impressum" title="Impressum"><i class="fas fa-info"></i></a>
-		    </li>
-			<li class="nav-item">
-		      <a class="nav-link p-2" href="datenschutz.html" rel="noopener" aria-label="Datenschutzerklärung" title="Datenschutzerklärung"><i class="fas fa-shield-alt"></i></a>
-		    </li>
-			<li class="nav-item">
-		      <a class="nav-link p-2" href="#" onclick="navigate(<?php echo htmlspecialchars($lat);?>,<?php echo htmlspecialchars($lon);?>);" rel="noopener" aria-label="Navigiere" title="Navigiere"><i class="fas fa-directions"></i></a>
-		    </li>
-			<li class="nav-item">
-		      <a class="nav-link p-2" href="#" onclick="timetable('<?php echo htmlspecialchars($countryCode);?>','<?php echo htmlspecialchars($stationId);?>','<?php echo htmlspecialchars($stationName);?>', '<?php echo htmlspecialchars($DS100);?>');" rel="noopener" aria-label="Abfahrtszeiten" title="Abfahrtszeiten"><i class="fas fa-list"></i></a>
-		    </li>
-			<li class="nav-item">
-		      <a class="nav-link p-2" href="#" onclick="providerApp('<?php echo htmlspecialchars($countryCode);?>');" rel="noopener" aria-label="Betreiber App" title="Betreiber App"><i class="fas fa-external-link-alt"></i></a>
-		    </li>
-	    </ul>
-	  </div>
-	</nav>
+<?php
+require_once("./navbar.php");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ["require_once" statement detected. File manipulations are discouraged. Statement is not a function, no parentheses are required.](https://app.codacy.com/app/RailwayStations/RSWebsite/pullRequest?prid=4676723)